File: speedtest01.c
// compile with:
// gcc speedtest01.c -o speedtest01_c.bin

#include <stdio.h>

#define END_LOOP 1000000000 // 1 Mrd.

void main() {
    unsigned int last_number=0;

    for (unsigned int i = 0; i < END_LOOP; i++) {
        last_number++;
    }
    printf("C:last_number=%d\n", last_number );
}

File: speedtest01.dart
const int END_LOOP = 1000000000; // 1 Mrd.

void main() {
  int lastNumber = 0;

  for (int i = 0; i < END_LOOP; i++) {
    lastNumber++;
  }
  print('Dart:last_number=$lastNumber\n--');
}
